src/node.h

```diff
@@ -528,22 +528,22 @@ NODE_EXTERN void AddPromiseHook(v8::Isolate* isolate,
 /* Returns the id of the current execution context. If the return value is
  * zero then no execution has been set. This will happen if the user handles
  * I/O from native code. */
-NODE_EXTERN async_uid AsyncHooksGetCurrentId(v8::Isolate* isolate);
+NODE_EXTERN async_uid AsyncHooksGetExecutionAsyncId(v8::Isolate* isolate);
 
-/* Return same value as async_hooks.triggerId(); */
-NODE_EXTERN async_uid AsyncHooksGetTriggerId(v8::Isolate* isolate);
+/* Return same value as async_hooks.triggerAsyncId(); */
+NODE_EXTERN async_uid AsyncHooksGetTriggerAsyncId(v8::Isolate* isolate);
```

Comment:
Should we start using `V8_DEPRECATED` in our headers to make note of deprecated native APIs?